bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/FileInfo.java

```diff
@@ -112,6 +114,29 @@ public long getSizeSinceLastwrite() {
         return sizeSinceLastwrite;
     }
 
+    public ByteBuffer getExplicitLac() {
```

Comment:
May be we should talk. I am not really sure with this explicitLac I am not sure if your change of caching will help in anyway. Basically I need  keep bytebuffer because it is checksum'd buffer and client need to read/check on the receive side. As I said in the past, I don't see the major advantage of caching piggyback LAC. If you agree, may be we can just live with explicitLAC. May be better thing is to have a call? and go over the code?